vmdb/spec/migrations/20150405141637_remove_port_config_from_container_service_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "spec_helper"
+require Rails.root.join("db/migrate/20150405141637_remove_port_config_from_container_service.rb")
+
+describe RemovePortConfigFromContainerService do
+  let(:container_service_stub)             { migration_stub(:ContainerService) }
+  let(:container_service_port_config_stub) { migration_stub(:ContainerServicePortConfig) }
+
+  migration_context :up do
+    it 'Moves port, protocol and host(target)-port to container service port configs table' do
+      service = container_service_stub.create!(:ems_ref        => "test_ref",
+                                               :name           => "test_service",
+                                               :protocol       => "TCP",
+                                               :port           => 1111,
+                                               :container_port => 2222)
+      migrate
+      pconfig = container_service_port_config_stub.where(:container_service_id => service.id).first
+      pconfig.protocol.should    == "TCP"
+      pconfig.port.should        == 1111
+      pconfig.target_port.should == "2222" # container_port:integer turns into target_port:string
```

Comment:
Sorry, i'm naive here, why is `target_port` ever `redis-server`  ???  It's strange to have numeric ports except when it's `redis-server`.